test/proposals/ProposalExecutionEngine.t.sol

```diff
@@ -212,6 +212,18 @@ contract ProposalExecutionEngineTest is Test, TestUtils {
         _executeProposal(executeParams);
     }
 
+    function test_executeProposal_isPayable() public {
+        uint256 emitValue = _randomUint256();
+        IProposalExecutionEngine.ExecuteProposalParams memory executeParams = _createTestProposal(
+            _createOneStepProposalData(emitValue)
+        );
+        vm.expectEmit(true, false, false, false, address(eng));
```

Comment:
Preferred to have all topics checked

Reply:
There's only one topic, but understand it's best practice. Will fix.